packages/db/src/models/granule.ts

```diff
@@ -20,15 +21,20 @@ export default class GranulePgModel extends BasePgModel<PostgresGranule, Postgre
     executionCumulusId: number,
     granulesExecutionsPgModel = new GranulesExecutionsPgModel()
   ) {
+    if (!granule.created_at) {
```

Comment:
So `created_at` is required because it's used in the query builder below now? But that's not true for a non-running granule. 

Also is an if/throw necessary if the query builder would fail without that field regardless?

Reply:
> But that's not true for a non-running granule.

Is there a legitimate case where created_at isn't being added in the reporting lambda prior to write?   We're requiring it as part of a workflow's `cumulus_meta` data, and it's added to all granule objects from there, unless I'm missing something.    

There's a default set to 'now'  on the table in case of manual insertion/some other update. 

This was more of a vanity error override but we should probably be doing a try/catch on the query-builder fail.  I can make that update.

Reply:
> Is there a legitimate case where created_at isn't being added in the reporting lambda prior to write? We're requiring it as part of a workflow's cumulus_meta data, and it's added to all granule objects from there, unless I'm missing something.

Yeah, we are always setting `created_at` for records written from the Lambda, or at least we should be. My point was more that the query builder for a non-running granule won't fail if `created_at` is missing. 

Was the idea of throwing the error here just to avoid a more confusing error thrown by the failed query?

I'm also not sure I understand the suggestion of adding a try/catch.

Reply:
> Was the idea of throwing the error here just to avoid a more confusing error thrown by the failed query? 

Yes, that.    

> I'm also not sure I understand the suggestion of adding a try/catch.

I'm suggesting that I could have the conversion method throw, and catch it rather than a preemptive if/throw.